docs/docs/deployments.md

```diff
@@ -5,118 +5,111 @@ title: Application Deployments
+- __minimumHealthCapacity == 0__ : All old instances can be killed before the new version is deployed.
+- __minimumHealthCapacity == 1__ : All instances of the new version are deployed side by side before the old version is stopped 
+- __minimumHealthCapacity between 0 and 1__ : Scale the old version to minimumHealthCapacity and start the new version to minimumHealthCapacity side by side. If this is completed successfully, the new version is scaled to 100% and the old version is stopped. 
 
-This gets a little bit more complex if there are dependencies.
-When the applications of the example above are updated, the following actions will be performed:
+This gets a bit more complex if there are dependencies.
+In the example above, when the applications are updated, Marathon performs the following actions:
   
-1. Upgrade application db until all instances are replaced, ready and healthy (taking upgradeStartegy into account)
-2. Upgrade application app until all instances are replaced, ready and healthy (taking upgradeStrategy into account)  
+1. Upgrade application _db_ until all instances are replaced, ready, and healthy (taking `upgradeStrategy` into account).
+1. Upgrade application _app_ until all instances are replaced, ready, and healthy (taking `upgradeStrategy` into account).
 
-Please take into account that your cluster needs to have more capacity available for the update process if you choose a minimumHealthCapacity greater 0.5.
-In this case more than half of the instances of the same application are run side by side.
-These capacity constraints are summed up if there are dependencies. In our example, we defined 0.6 for db and 0.8 for app. 
-This means that in the update case, we have 12 instances of db (6 old and 6 new) and 32 instances of app (16 old and 16 new) running side by side.
+**Note:** Your cluster needs to have more capacity available for the update process if you choose a `minimumHealthCapacity` greater 0.5. When `minimumHealthCapacity` is greater than 0.5, more than half of the instances of the same application are run side by side.
+These capacity constraints are added together if there are dependencies. In our example, we defined 0.6 for _db_ and 0.8 for _app_. 
+This means that when an update is performed there will be 12 instances of _db_ (6 old and 6 new) and 32 instances of _app_ (16 old and 16 new) running side by side.
 
 ## Force a Deployment
 
 An application can be changed by only one deployment at a time.
 Other changes to the application must wait until the first deployment has finished.
-It is possible to break this rule by running a deployment with the force flag.
-The REST interface allows the force flag for all state-changing operations.
+You can break this rule by running a deployment with the `--force` flag.
```

Comment:
why is `force` prefixed with `--` here? do you have some CLI in mind? Marathon doesn't ship with a CLI out of the box. Pretty sure the original meaning of "flag" here was referring to the use of `force=true` as an API call parameter.